scraper/src/strategies/default_strategy.py

```diff
@@ -213,10 +213,12 @@ def get_records_from_dom(self, current_page_url=None):
             # http://localhost:1314/index.html#pyansys-actions
             cname = os.getenv("DOCUMENTATION_CNAME")
             if cname is not None:
-                http, _, port_and_page = record["url"].split(":", 2)
-                port = os.getenv("DOCUMENTATION_PORT")
-                path = port_and_page[len(port):]
-                record['url'] = f"{http}s://{cname}{path}"
+                url_parts = record.get("url", "").split(":", 2)
+                if len(url_parts) > 2:
+                    http, _, port_and_page = url_parts
+                    port = os.getenv("DOCUMENTATION_PORT")
+                    path = port_and_page[len(port):]
+                    record['url'] = f"{http}s://{cname}{path}"
```

Comment:
```suggestion
                http, _, port_and_page = record.get("url", "").split(":", 2)
                port = os.getenv("DOCUMENTATION_PORT")
                path = port_and_page[len(port):]
                record['url'] = f"{http}s://{cname}{path}"
```

Reply:
@Revathyvenugopal162 - revert your change. split(":", 2) is ensuring that you will always get three elements back as max. Which is what we need. No need to verify it with an if statement.